app/v2/controllers/V2MovementsController.scala

```diff
@@ -122,7 +123,7 @@ class V2MovementsControllerImpl @Inject() (
     }
 
   private def submitDepartureDeclarationXML(): Action[Source[ByteString, _]] =
-    (authActionNewEnrolmentOnly andThen messageSizeAction()).stream {
+    (authActionNewEnrolmentOnly andThen acceptHeaderActionProvider(isOnlyJson) andThen messageSizeAction()).stream {
```

Comment:
instead of setting a val to isOnlyJson and passing it or !isOnlyJson, can we just pass the boolean through instead please?

you can specify the parameter in the call to make what you're passing more intuitive:

`acceptHeaderActionProvider(isOnlyJson = true)`

Reply:
Updated with the change mentioned above